packages/mds-compliance-service/repository/entities/compliance-snapshot-entity.ts

```diff
@@ -63,6 +63,6 @@ export class ComplianceSnapshotEntity
   @Column('jsonb')
   vehicles_found: ComplianceSnapshotEntityModel['vehicles_found']
 
-  @Column('jsonb')
+  @Column('jsonb', { default: '[]' })
```

Comment:
This missing column option causes unnecessary `SET/DROP DEFAULT` statements in all new `mds-compliance-service` migrations.